### ISSUE TITLE
Reverse Mtime Sorting

### ISSUE THREAD

**Issue by Gajus84** [phillipberndt/pqiv#191]
Hi, I'm really enjoying your program. There is one thing though, that bothers me a little:

Having Mtime as a sorting option sorts my pictures from old to new in montage mode, but often I would like to see my newest pictures first, especially when listing folders from several years. 

Would there be a possibility to implement such a feature or am I just missing something and this is already possible?

Thank you and best regards.

**Comment by phillipberndt** [comment]
Makes sense and isn't a lot of work. Have you considered just swapping the keys for next and previous? Happy to add this, I just thought I'd better ask whether this'd suffice..

**Comment by Gajus84** [comment]
On a grid I would find this slightly irritating. You press left and your selector moves right.
My "workaround" for now is just pressing "End" and work from there upwards.

This is by no means an important issue. Just a "you've got your "perfect" setup, and only one little thing isn't as you'd like" -thing.
